pool/test/tenant_test.ml

```diff
@@ -81,7 +81,7 @@ module Data = struct
     let server = "smtp.uzh.ch"
     let port = 587
     let username = "engineering@econ.uzh.ch"
-    let password = "emailemail"
+    let password = "Password!"
```

Comment:
Number is missing

Reply:
😄 That is the smtp password. I resented it to the old one.

I don't think we need a password policy for smtp passwords. That should be part of the email service, shouldn't it?